### PR TITLE
[issue-#6] Include flag as is (no case formatting)

### PR DESCRIPTION
**Pull request by tboulis**
Added optional second parameter to `mockFlags()` function that is of type:
```
type mockFlagsOptions = {
  skipFormatting?: boolean
}
```
If skipFormatting is true, the flags will be mocked without formatting them to camelCase and kebab-case.

Related to https://github.com/launchdarkly-labs/jest-launchdarkly-mock/issues/6

### EDIT #1:
*Based on code review comment, the original flag is added as-is so the pristine value is returned*

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -37,15 +41,21 @@ export const ldClientMock = {
...
 
-export const mockFlags = (flags: LDFlagSet) => {
+export const mockFlags = (flags: LDFlagSet, options?: mockFlagsOptions) => {
   mockUseFlags.mockImplementation(() => {
     const result: LDFlagSet = {}
-    Object.keys(flags).forEach((k) => {
-      const kebab = kebabCase(k)
-      const camel = camelCase(k)
-      result[kebab] = flags[k]
-      result[camel] = flags[k]
```
[the remark this answers is not in the document]
Comment (yusinto) [reply src/index.ts:63]: Thank you @tboulis ! I'll go ahead and merge this now. I will make a release soon. Thanks again.

File: README.md [hunk 2]
```diff
@@ -32,7 +32,11 @@ module.exports = {
 ## Usage
 Use the only 3 apis for test cases:
 
-* `mockFlags(flags: LDFlagSet)`: mock flags at the start of each test case.
+* `mockFlags(flags: LDFlagSet, Options?: mockFlagsOptions)`: mock flags at the start of each test case. 
+###### Options
+| name | type | default | description |
+| ---- | ---- | ------- | ----------- |
+| skipFormatting(optional) | boolean | false | If true, flags will not be formatted to camelCase or kebab-case |
```
Comment (yusinto) [inline README.md:39]: I think this change can be removed now? It is no longer relevant with your change below.

```suggestion
`mockFlags(flags: LDFlagSet)`: mock flags at the start of each test case.
```

File: src/index.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (yusinto) [reply src/index.ts:63]: One more [comment](https://github.com/launchdarkly-labs/jest-launchdarkly-mock/pull/7/files#r795987675) before I can merge this @tboulis please. Thank you.

File: README.md (hunk 2 above)
Comment (tboulis) [reply README.md]: Yes, sorry for missing this

File: src/index.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (tboulis) [reply src/index.ts:63]: Sure @yusinto !

**Review by yusinto — APPROVED** [review]

**Comment by yusinto** [comment]
This has been [released](https://github.com/launchdarkly-labs/jest-launchdarkly-mock/releases/tag/v.0.3.1). Thanks @tboulis

**Pull request merged**